tests/test_parser_utils.py

```diff
@@ -0,0 +1,367 @@
+"""Test the CWL parsers utility functions."""
+
+from _pytest.python_api import raises
```

Comment:
Why the private import?

Reply:
Because PyCharm for some unknown reason keeps suggesting me that as the first option an I sometimes just click Return :( Let me fix it